src/api/payment-requests/payment-requests.md

```diff
@@ -22,9 +22,10 @@ Payment Requests have the following statuses:
 
  * **new**: after being created.
  * **paid**: after being paid with one or more transactions.
- * **cancelled**: after being cancelled or voided by the merchant.
+ * **cancelled**: after being cancelled by the merchant.
  * **expired**: after expiry time is reached without being paid or cancelled.
  * **refunded**: after being partially or fully refunded.
+ * **voided**: after being voided by the merchant.
```

Comment:
See explanation [here](https://github.com/centrapay/centrapay-docs/pull/435#discussion_r890638029).

Reply:
Lets put this back, we've talked this over and it's not a new status on the payment request that we're after. That discussion was around void payment activities being used to support legacy endpoints.